api_server/src/lib.rs

```diff
@@ -137,6 +141,9 @@ impl ApiServer {
             })
             .map_err(Error::Io);
 
+        // Load seccomp filters on the API thread.
+        apply_seccomp_func();
```

Comment:
Just wondering, since the `api` crate has a `vmm` dependency anyway, can't we just use the `vmm::default_syscalls::set_seccomp_level` function here, instead of getting something equivalent in the closure?